src/mappings/TinlakeClaimRAD.ts

```diff
@@ -1,5 +1,5 @@
 import { log, BigDecimal } from '@graphprotocol/graph-ts'
-import { Claimed } from '../../generated/Claim/TinlakeClaimRAD'
+import { Claimed } from '../../generated/Claim/TinlakeClaimRad'
```

Comment:
should we rename the file we're in as well? `mappings/TinlakeClaimRad.ts`

Reply:
Yes, maybe! I was trying to be mininal, but I agree it would be more clean to rename everywhere

Reply:
👍